tools/github-actions/cascading/readme.md

```diff
@@ -0,0 +1,29 @@
+# Cascading action
+## Overview
+This plugin is meant to replace the Bitbucket cascading mechanism on Azure Dev Ops.
+For the moment, the branching model supported is the following one : release/Maj.min[.0-alpha|beta|next|rc] and a default branch from your choice.
+It has been restrained as much as possible for the first version, to avoid any mistakes.
+
+## Task options
+See [Action specifications](action.yml) directly for more information about the supported parameters.
+
+
+## What it does
+The plugin will look for branches matching release/* in your repository.
+Then the plugin will sort the branches, and based on the current branch you gave as input will figure out if the cascading is needed and on which branch.
+If the current branch is the default branch given as input OR if the current branch is the last one matching the pattern the plugin will do nothing.
+
+If a branch candidate for cascading is found, a git merge attempt will be performed :
```

Comment:
If the plugin finds a branch candidate for cascading, it will try a git merge and:
(always prefer active voice for documentation)